src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -545,7 +545,12 @@ private async Task<bool> ScanTargetsAsync(IEnumerable<Skimmer<TContext>> skimmer
                     {
                         context = _fileContexts[item];
 
-                        DetermineApplicabilityAndAnalyze(context, skimmers, disabledSkimmers);
+                        rootContext.Logger.AnalyzingTarget(context);
```

Comment:
>AnalyzingTarget [](http://example.com/codeflow?start=43&length=15)

It seems off you have to introduce this, why is that? weren't we calling into analyzing target before?

Reply:
I have debugged, it is only logged once so it was not called before.

Reply:
I think you are saying we had a bug before. did you author a unit test that flags the problem and which then passes thanks to your change?